todo_app/app.py

```diff
@@ -1,44 +1,69 @@
+from datetime import date
+from os import stat
 from flask import Flask, render_template, redirect, request
 
-from todo_app.data.session_items import *
+from todo_app.data.trello_items import *
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
-def index():    
-    list = sorted(get_items(), key=lambda item:item['status'], reverse=True)
-    return render_template("index.html", to_do_items=list)
+def index():
+    sorted_list = sorted(get_tasks(), key=lambda item:item.status, reverse=True)
+    number_of_tasks = sorted_list.count
+    return render_template("index.html", to_do_items=sorted_list, empty_to_do_list=number_of_tasks)
+
+@app.route('/<status>')
+def filtered_index(status):
+    status_filter = ''
+    match(status):
```

Comment:
this `match` syntax was only added in python 3.10. This is fine by itself, but you should probably update the readme (as it mentions only requiring python 3.7+) and update your `pyproject.toml` to require `python = "^3.10"`